### PR TITLE
convert to origami 2.0 component

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -14,12 +12,11 @@
...
 	},
 	"supportStatus": "active",
-	"keywords": "pagination",
 	"demosDefaults": {
 		"sass": "demos/src/demo.scss",
 		"dependencies": [
-			"o-fonts@^4.0.0",
-			"o-normalise@^2.0.0"
+			"@financial-times/o-fonts@5.0.0-1",
+			"@financial-times/o-normalise@3.0.0-0"
```
Comment (JakeChampion) [inline origami.json:19]:
```suggestion
			"o-fonts@5.0.0-1",
			"o-normalise@3.0.0-0"
```

File: .github/workflows/test-origami-component.yml [hunk 2]
```diff
@@ -5,18 +5,11 @@ jobs:
...
-      id: repo_name
-      run: echo ::set-output name=REPO_NAME::${GITHUB_REPOSITORY/*\//}
-    - run: npx origami-ci branch
-      env:
-        ORIGAMI_CI_NAME: ${{ steps.repo_name.outputs.REPO_NAME }}
-        ORIGAMI_CI_VERSION: 0.0.0
-        GITHUB_TOKEN: ${{ secrets.ORIGAMI_FOX_ACCESS_TOKEN }}
-        ORIGAMI_PULL_REQUEST: ${{ github.event.pull_request.html_url }}
+    - run: npm install -g npm@^7
+    - run: npm install -g origami-build-tools@prerelease
```
Comment (chee) [inline .github/workflows/test-origami-component.yml:12]:
```suggestion
    - run: npm install -g origami-build-tools@11
```

File: package.json [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
+	],
+	"homepage": "https://registry.origami.ft.com/components/o-buttons",
+	"bugs": {
+		"url": "https://registry.origami.ft.com/components/o-buttons"
+	},
+	"license": "MIT",
+	"type": "module",
+	"scripts": {
+		"start": "npx serve ./demos/local",
+		"build": "npm_config_yes=true npx \"origami-build-tools@prerelease\" install && npm_config_yes=true npx \"origami-build-tools@prerelease\" demo"
```
Comment (chee) [inline package.json:16]:
```suggestion
		"build": "npm_config_yes=true npx \"origami-build-tools@11\" install && npm_config_yes=true npx \"origami-build-tools@11\" demo"
```

File: package.json [hunk 4]
```diff
@@ -1,18 +1,41 @@
...
 	},
-	"scripts": {
-		"start": "npx serve ./demos/local",
-		"build": "npm_config_yes=true npx \"origami-build-tools@^10\" install && npm_config_yes=true npx \"origami-build-tools@^10\" demo"
+	"peerDependencies": {
+		"@financial-times/o-brand": "^4.0.2",
+		"@financial-times/o-colors": "^6.0.1",
+		"@financial-times/o-icons": "^7.0.0",
+		"@financial-times/o-normalise": "^3.0.0",
+		"@financial-times/o-typography": "^6.4.6"
```
Comment (JakeChampion) [inline package.json:32]: this looks wrong, I think it is meant to be ^7

**Comment by JakeChampion** [comment]
@Financial-Times/origami-core this is now ready for a review

**Review by chee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v7.0.0 :tada:

**Pull request merged**